src/tutorial/solvers.md

```diff
@@ -152,7 +152,7 @@ each of these examples in more details below.
 
 - Try running `./ex2p` to run a 2D elasticity problem.
 
-- As in example 1, the linear system is solved using _algebraic multigrid_.
+- As in Example 1, the linear system is solved using AMG.
 
 - For this example, two types of AMG solvers can be used:
 
```

Comment:
GitHub won't let me comment exactly in these places:
Lines 164-167: Where in `ex2p` do I add the `-elast` flag or make the `--order` argument? Or do I change something in the command line?
Lines 230-238: Same as the question above, where do I make these changes? I would like to see additional sample run(s) or lines of code that I'm supposed to change as in 198-199 and 284-185.